src/models/wikimedia/wikipedia/analyzer.py

```diff
@@ -127,10 +127,11 @@ def __gather_reference_statistics__(self):
             for reference in self.article.extractor.references:
                 if not reference:
                     raise MissingInformationError("raw_reference was None")
-                if reference.footnote_subtype:
-                    subtype = reference.footnote_subtype.value
-                else:
-                    subtype = ""
+                subtype = (
+                    reference.footnote_subtype.value
+                    if reference.footnote_subtype
+                    else ""
+                )
```

Comment:
I do know if these are an improvement or not...
% `ruff rule SIM118`
ruff rule SIM108
# if-else-block-instead-of-if-exp (SIM108)

Derived from the **flake8-simplify** linter.

Autofix is sometimes available.

Message formats:
* Use ternary operator `{contents}` instead of `if`-`else`-block